docs/snippets/edit-this-page.md

```diff
@@ -0,0 +1,35 @@
+<style>
+.contribution-button {
+    border-radius: 5px;
+    background-color: #EFEFEF;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-top: 5px;
+    padding-bottom: 5px;
+    color: black;
+    weight: bolder;
+    display: inline-flex;
+    margin-top: -10px;
+    align-items: center;
+    border: 1px solid #CCCCCC;
+    text-decoration: none;
+}
+.contribution-button:hover {
+    background-color: #DDDDDD;
+    cursor: pointer;
+    color: #000000;
+}
+</style>
+<br/><br/>
+
+----
+
+#### Help Us Make This Page Better
+
+All our docs are open source. If something is wrong, or unclear, submit a PR to fix it!
+
+<a class="contribution-button" style="text-decoration:none" href="!!LINK!!" target="_blank">
+    <img src="https://raw.githubusercontent.com/primer/octicons/main/icons/git-pull-request-16.svg" width="16" height="16" style="display: inline-block; vertical-align: middle;" alt="Pull request icon">
+<span style="margin-left: 5px;">Make a Contribution</span></a>
```

Comment:
I wonder if this should just say "Contribute." It seems more concise, but that's just personal preference.